### ISSUE TITLE
Some PV data is NaN

### ISSUE THREAD

**Issue by JackKelly** [openclimatefix/nowcasting_dataset#622]
In v15 of the dataset, batch `000000.nc`, example index 27, pv system index 64, all the timeseries values are NaN:

```
pv_batch[27, :, 65]
array([nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan,
       nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan, nan,
       nan, nan, nan, nan, nan])
```

But this isn't a "missing" PV system.

**Comment by peterdudfield** [comment]
Would you suggest we filter out any systems that have capacity of zero? 

Another way is to use normalization code somewhere that takes this into account. 
For example could add a line of code in https://github.com/openclimatefix/nowcasting_dataloader/blob/main/nowcasting_dataloader/data_sources/pv/pv_model.py#L137 - i know however you arent using the dataloader at the moment

**Comment by JackKelly** [comment]
> Would you suggest we filter out any systems that have capacity of zero?

Yeah, exactly that! Let's throw away any PV systems with a capacity of zero. :slightly_smiling_face:
